src/main/kotlin/com/feedback/dao/Models.kt

```diff
@@ -0,0 +1,87 @@
+package com.feedback.dao
+
+import com.hibob.nullability_exercises.nullSafeToUpper
+import java.sql.Timestamp
+
+data class RetrieveEmployeeRequest(
```

Comment:
This is not the best name. The "request" was just the id, and then you got the employee details. So maybe rename it to something like `PersistedEmployee`, or just `Employee`. `PersistedEmployee` implies that you fetched it from the DB. Rename the other places as well